packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -490,6 +490,26 @@ export class StripeWebhookHandler extends StripeHandler {
         customer,
         subscription
       );
+
+      if (
+        account &&
+        subscription.metadata['redundantCancellation'] === 'true'
+      ) {
+        const invoiceDetails =
```

Comment:
nit. `subscriptionDetails` since it gets the details of a specific subscription?